tests/test_cases.rs

```diff
@@ -0,0 +1,88 @@
+use zram_generator::{config, generator};
+
+use anyhow::Error;
+use tempfile::TempDir;
+use std::fs;
+use std::path::Path;
+use std::process::Command;
+
+fn prepare_directory(srcroot: &Path) -> Result<TempDir, Error> {
```

Comment:
I think easier to use Result from anyhow, so it would look just like `-> Result<TempDir>`